lib/src/test/kotlin/starknet/data/TypedDataTest.kt

```diff
@@ -548,7 +548,7 @@ internal class TypedDataTest {
                 "name": "DomainV1",
                 "version": "1",
                 "chainId": "2137",
-                "revision": "1"
```

Comment:
do we have a test with "1" as well? I suppose it may still be possible to have such response, right?

Reply:
Correct, considering `"1"` may occur in response, then these tests should be also added.

Reply:
I don't think `"1"` is possible per the https://github.com/starknet-io/SNIPs/pull/79

Reply:
> but this is atm limited to revision 0 right? 

Well yeah, but also:

> - **revision (optional)**: the revision of the specification to be used. If the value is omitted it will default to `0` .
>     - Revision `0`: Represents the de facto spec before this SNIP was published. The purpose is to help with backwards compatibility. It’s not recommended to use it.

We can add the test just for the sake of it, but we're testing something very unlikey

> I find this a bit confusing :D

It is, but apparently handling shortstrings as ints or vice versa is a typical thing for typed data as I previously discovered trying to introduce some additional type checks which resulted in tests failing for existing examples